tests/plotting/test_actor.py

```diff
@@ -233,6 +234,21 @@ def test_vol_actor_prop(vol_actor):
     assert vol_actor.prop is prop
 
 
+@pytest.mark.parametrize('func', [np.ndarray, scipy.spatial.transform.Rotation.from_matrix])
+def test_rotation_from(actor, func):
+    array = [
+        [0.78410209, -0.49240388, 0.37778609],
+        [0.52128058, 0.85286853, 0.02969559],
+        [-0.33682409, 0.17364818, 0.92541658],
+    ]
+
+    actor.rotation_from(array)
```

Comment:
Whoops, this is a key part of the test and `scipy` is not used/tested otherwise.
```suggestion
    rotation = func(array)
    actor.rotation_from(rotation)
```